platform/spec/services/temporal/activities/convert_read_record_activity_spec.rb

```
@@ -0,0 +1,145 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe Temporal::Activities::ConvertReadRecordActivity do
+  let(:activity) { described_class.new(double("context")) }
```

rubocop (1.68.0)

[convention] Prefer using verifying doubles over normal doubles.

(RSpec/VerifiedDoubles)